hazelcast-jet-sql/src/main/java/com/hazelcast/jet/sql/impl/type/QueryDataTypeUtils.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.sql.impl.type;
+
+import com.hazelcast.sql.impl.type.QueryDataType;
+import com.hazelcast.sql.impl.type.QueryDataTypeFamily;
+
+public final class QueryDataTypeUtils {
+
+    private QueryDataTypeUtils() {
+    }
+
+    public static String sqlTypeName(QueryDataType type) {
```

Comment:
Note that in IMDG we currently follow the rules below:
1) Integer type is reported as `INTEGER`, not `INT`
2) Timestamp with time zone is reported as `TIMESTAMP_WITH_TIME_ZONE`

We do not have DML/DDL commands in IMDG now, so we are not concerned with this naming on the parser level. However, these names are reported in query metadata and error messages. Going forward, we are going to utilize them on the parser level as well. The canonical type names are enum names from `SqlColumnType`.

I would suggest you consider following the common naming scheme here, otherwise, users might be confused when to and how to use type names.

Reply:
This method is also used in `unparse()` where it should produce text that's parseable. And in IMDG you already use `CAST(a AS INT)`, not `AS INTEGER`. It's also used in `information_schema.columns.data_type` - here we could use INTEGER. Also in one error message.

I don't understand why we actually have this dichotomy: INT and INTEGER... I think having one would be the least confusing.

Reply:
Unparse is largely unused in IMDG (at least I am not aware of when it might be needed and exposed to the user). 

What I am saying that the current decision in IMDG is to display 4-byte type as `INTEGER` everywhere. There could be possible places where we missed it, but still. In tests we also use `as INTEGER`, e.g. `CastIntegrationTest.testInteger`

Reply:
Refactored the code that it's only used in the `unparse`. We have a method to recreate the DDL command from an existing mapping. It's not used in this PR, but I'd like to keep the code there.